napari/_qt/menus/_tests/test_plugins_menu.py

```diff
@@ -31,7 +31,7 @@ def widget2():
     assert len(viewer.window._dock_widgets) == 0
     plugin_action_menu.actions()[0].trigger()
     assert len(viewer.window._dock_widgets) == 1
-    assert list(viewer.window._dock_widgets.data)[0] == 'Widget 1 (tmp_plugin)'
```

Comment:
That's... wierd to me. What rule is this? o.o

Reply:
https://beta.ruff.rs/docs/rules/unnecessary-iterable-allocation-for-first-element/

Reply:
that's pretty clever!

Reply:
makes sense I guess :P